pkg/stream/input_sqs.go

```diff
@@ -212,6 +212,10 @@ func (i *sqsInput) AckBatch(ctx context.Context, msgs []*Message) error {
 	return multiError.ErrorOrNil()
 }
 
+func (i *sqsInput) HasRetry() bool {
```

Comment:
Is there a designed way to overwrite this? In case of batch consumption, it's beneficial to not use the queue's retry mechanism but a custom one, so we can acknowledge all messages about reading.

In automation we sometimes have the problem that processing a message takes such a long time that it's already redelivered by the primary queue as we acknowledge it only after successful processing.

Reply:
This sounds for me like an additional PR make the ack behaviour configurable?